content/en/docs/setup/keadm.md

```diff
@@ -105,12 +157,13 @@ Run `keadm gettoken` in **cloud side** will return the token, which will be used
 
 ### Join Edge Node
 
-`keadm join` will install edgecore and mqtt. It also provides a flag by which a specific version can be set.
+#### keadm join
+`keadm join` will install edgecore and mqtt. It also provides a flag by which a specific version can be set. It will pull image `kubeedge/installation-package` from dockerhub and copy binary `edgecore` from container to hostpath, and then start `edgecore` in binary progress.
```

Comment:
```suggestion
`keadm join` will install edgecore. It also provides a flag by which a specific version can be set. It will pull image `kubeedge/installation-package` from dockerhub and copy binary `edgecore` from container to hostpath, and then start `edgecore` as system service.
```